packages/core/content-type-builder/admin/src/components/IconPicker/index.js

```diff
@@ -0,0 +1,201 @@
+import React, { useState, useRef } from 'react';
+import {
+  Box,
+  Flex,
+  Icon,
+  Typography,
+  Searchbar,
+  IconButton,
+  inputFocusStyle,
+  VisuallyHidden,
+  Field,
+  FieldLabel,
+  FieldInput,
+} from '@strapi/design-system';
+import { Trash, Search } from '@strapi/icons';
+import { useIntl } from 'react-intl';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+import { getTrad } from '../../utils';
```

Comment:
This is really nit-picky, but could you please but an empty lines before this import to separate locale from global imports?

Reply:
We can do an eslint rule for this you know – then just run autofix 👀

Reply:
That would be amazing.

Reply:
@gu-stav could you share why this would be better? I tend to naturally just ignore the imports when I'm reading/scanning a code file, and splitting imports in 2 blocks makes that harder

Reply:
@remidej It really comes down to preference. Let's discuss it in the next FE sync and just add an eslint rule for that and we don't have to discuss it ever again :)